lib/src/components/NxLoadError/NxLoadError.scss

```
@@ -0,0 +1,21 @@
+.nx-load-error__message {
+  // TODO no magic numbers
+  max-width: calc(100% - 32px);
```

```suggestion
  max-width: calc(100% - 32px);
```
  margin-bottom: $nx-spacing-sm;
  max-width: calc(100% - 32px);